sources/lib/util/options.ml

```diff
@@ -148,6 +155,13 @@ let vsave_used_context = ref false
 let vprofiling_period = ref 0.
 let vprofiling = ref false
 
+let match_format f =
+  match f with
+  | "native" | "ae" | "why" | "mlw" -> "native"
+  | "smtlib" | "smt" | "smt2" | "psmt" | "psmt2" -> "smtlib"
+  | "why3" | "Why3" -> "why3"
+  (* | "szs" | "SZS" -> "szs" *)
+  | s -> s
```

Comment:
With the split of type between input format and output format, this function will also need to be split, and in that case please differentiate the following things:
- the format itself
- the unique and canonic name of the a given input or output format (i.e. `smtlib2` (the `2` here is important because version 3 is coming with a change in semantics as I understand it), `tptp`, `alt-ergo`, etc..)
- the extension(s) supported by a given format.
- optionally (though I'd prefer if there was none), the aliases recognised to be the same as a format's canonic name.